test/v1/models/response/amendPeriodSummary/AmendPeriodSummaryResponseSpec.scala

```diff
@@ -16,8 +16,9 @@
 
 package v1.models.response.amendPeriodSummary
 
+import api.hateoas
```

Comment:
Remove import for this file and :

- test/v1/controllers/RetrievePeriodSummaryControllerSpec.scala
- test/v1/models/response/amendSEAnnual/AmendAnnualSubmissionResponseSpec.scala
- test/v1/models/response/createPeriodSummary/CreatePeriodSummaryResponseSpec.scala
- test/v1/models/response/listPeriodSummaries/ListPeriodSummariesResponseSpec.scala
- test/v1/models/response/retrieveAnnual/RetrieveAnnualSubmissionResponseSpec.scala
- test/v1/models/response/retrievePeriodSummary/RetrievePeriodSummaryResponseSpec.scala
- test/v2/controllers/AmendPeriodSummaryControllerSpec.scala
- test/v2/controllers/CreatePeriodSummaryControllerSpec.scala
- test/v2/controllers/RetrievePeriodSummaryControllerSpec.scala
- test/v2/models/response/amendPeriodSummary/AmendPeriodSummaryResponseSpec.scala
- test/v2/models/response/retrievePeriodSummary/RetrievePeriodSummaryResponseSpec.scala